docs/ref/contrib/postgres/operations.txt

```diff
@@ -35,6 +35,16 @@ the ``django.contrib.postgres.operations`` module.
     Installs the ``hstore`` extension and also sets up the connection to
     interpret hstore data for possible use in subsequent migrations.
 
+``CITextExtension``
+===================
+
+.. class:: CITextExtension()
+
+    .. versionadded:: 1.11
+
+    A subclass of :class:`~django.contrib.postgres.operations.CreateExtension`
```

Comment:
Chop "A subclass of :class:`~django.contrib.postgres.operations.CreateExtension` which will" (this was done for the other extensions since you started the PR probably).